user/customizing-the-build.md

```diff
@@ -454,13 +454,15 @@ script: ./test.py $TEST_SUITE
 creates a build matrix with 3 jobs, which runs test suite for each version
 of Python.
 
-#### Explicitly Included Jobs need complete definitions
+#### Explicitly Included Jobs inherits the first value in the array
 
-When including jobs, it is important to ensure that each job defines a unique value
-to any matrix dimension that the matrix defines.
+The jobs which are explicitly included inherits the first value of the expansion
+keys defined.
 
-For example, with a 3-job Python build matrix, each job in `matrix.include` must also
-have the `python` value defined:
+In this example with a 3-job Python build matrix, each job in `matrix.include`
+has the `python` value set to `'3.5'`.
+
+If this is not the desired behavior, override it in the job definition.
```

Comment:
Can we clarify this a little, the 'this' 'it' 'the' are confusing. Maybe:

> In this example with a 3-job Python build matrix, each job in `matrix.include`
 has the `python` value set to `'3.5'`. You can explicitly set the python version for a specific entry: